packages/plugins/FileService/src/types.ts

```diff
@@ -15,3 +48,18 @@ export interface FileInfo {
 export type FileInfoV1 = Omit<FileInfo, 'type' | 'provider'> & {
     type: 'arweave'
 }
+
+export interface JWKPublicInterface {
```

Comment:
Hmm I mean, there is a built-in type you can use without definition called `JsonWebKey`. Isn't that enough?

Reply:
yeah, I understand, but `JsonWebKey ` can't pass the type check.

``` typescript
import type { JWKInterface } from 'arweave/web/lib/wallet'
```